examples/angular_bazel_architect/package.json

```diff
@@ -3,11 +3,11 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
-    "build": "bazel build :build",
-    "test": "bazel test :test",
-    "lint": "ng lint",
-    "e2e": "ng e2e",
+    "start": "bazelisk run :serve",
+    "build": "bazelisk build :build",
+    "test": "bazelisk test :test",
+    "lint": "bazelisk run :lint",
```

Comment:
Should actually this be a using test so that it is not run for unchanged sources?

Reply:
I have a whole design for running lints. I don't think it should be a test because `bazel test //...` shouldn't fail just because you have something to fix before merging your change - basically an opinion on the developer workflow. Instead you use an Aspect (or an --experimental_extra_action_listener) so that you have a command-line way to say "re-run a portion of the build, but when you visit a ts_library target, run this other thing on its inputs". This way you get the benefits of caching and incremental re-lint but it's still a sidecar workflow.
Let's discuss separately?

Reply:
Yes, let’s take this offline.